### PR TITLE
Work on improvements for ui.operatingsys and ui.partitiontable

### PR DESCRIPTION
**Pull request by oshtaier**
Added more improvements for UI part corresponding to #2562 issue and our style in general. Fixed critical bug in logic for `test_set_domain_parameter_negative_2` and `test_negative_set_os_parameter_same_values` methods
```
nosetests tests/foreman/ui/test_operatingsys.py
....................S.........................
----------------------------------------------------------------------
Ran 46 tests in 1228.700s

OK (SKIP=1)
```
```
nosetests tests/foreman/ui/test_partitiontable.py
.....................EE.E..S..
======================================================================
FAILED (SKIP=1, errors=3)
```
These 3 errors related to my local environment issues with alerts

### CODE REVIEW

File: tests/foreman/ui/test_operatingsys.py [hunk 1]
```diff
@@ -79,19 +72,23 @@ def test_positive_create_os(self, test_data):
...
-            make_os(session, name=test_data['name'],
+            make_os(session,
+                    name=name,
                     major_version=test_data['major_version'],
                     minor_version=test_data['minor_version'],
                     description=test_data['desc'],
-                    os_family=test_data['os_family'], archs=[arch])
+                    os_family=test_data['os_family'],
+                    archs=['i386'],
+                    )
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_operatingsys.py]: Nitpick:

```python
            make_os(
                session,
                name=name,
                major_version=test_data['major_version'],
                minor_version=test_data['minor_version'],
                description=test_data['desc'],
                os_family=test_data['os_family'],
                archs=['i386'],
            )
```

**Comment by Ichimonji10** [comment]
Nice job. ACK. Please fix the nitpick if possible, but don't block on it.

File: tests/foreman/ui/test_operatingsys.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/ui/test_operatingsys.py]: :+1:

**Comment by elyezer** [comment]
ACK

**Pull request merged**